checker/tests/nullness/java8inference/Issue980.java

```diff
@@ -24,6 +24,8 @@ void m(List<String> strings) {
         // This assignment issues a warning due to incompatible types:
```

Comment:
Related to your comment below: why is there no expected warning about these incompatible types?